docs/frequently-asked-questions.rst

```diff
@@ -230,20 +205,6 @@ Are comments included with deployed contracts and do they increase deployment ga
 No, everything that is not needed for execution is removed during compilation.
 This includes, among others, comments, variable names and type names.
 
-What happens if you send ether along with a function call to a contract?
-========================================================================
-
-It gets added to the total balance of the contract, just like when you send ether when creating a contract.
-You can only send ether along to a function that has the ``payable`` modifier,
```

Comment:
Is this sentence covered elsewhere?

Reply:
@axic I feel like it's covered in several places already, search for `payable` in the docs and let me know if you agree.